context.go

```diff
@@ -86,4 +86,4 @@ func (c *Context) run(language, code, stdinGlob string) (string, Message) {
 }
 
 // Languages returns a list of available language names.
-func (c *Context) Languages() []string { return c.compilers.availableLanguages() }
+func (c *Context) Languages() map[string]CompositionDetails { return c.compilers.availableLanguages() }
```

Comment:
It might be nice to expose endpoints that return both the raw list of languages and the composition details.